Header header
float64 acc
float64 steer_rate

# if open_loop is set, use the state as desired state 
# and omit the control loop
std_msgs/Bool is_openloop
State state





